bitbucket/bb-pipeline-awscli/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM atlassian/default-image:2
 
-RUN apt-get update && apt-get install -y \
+RUN npm install -g npm@latest && apt-get update && apt-get install -y \
```

Comment:
Atlassian hasn't provided an updated base image so this is the hope and dream - to update npm and get npm ci capabilities along with the possibility of fixing some strange snyk issues.